scripts/e2e-ci.sh

```diff
@@ -296,6 +296,23 @@ test_phased_train() {
     task Multi-phase training Complete
 }
 
+test_phased_train_resume() {
+    task Train the model with LAB multi-phase training resume
+
+    # 'Declare and assign separately to avoid masking return values' <-- error.
```

Comment:
I don't quite understand this comment

Reply:
It's copied from test_phased_train.  You get a shell linting error if the local isn't defined separate from the assignment.